libraries/core_libs/consensus/src/vote_manager/vote_manager.cpp

```diff
@@ -21,6 +21,7 @@ VoteManager::VoteManager(const addr_t& node_addr, std::shared_ptr<DbStorage> db,
       final_chain_(std::move(final_chain)),
       next_votes_manager_(std::move(next_votes_mgr)) {
   LOG_OBJECTS_CREATE("VOTE_MGR");
+  period_ = pbft_chain_->getPbftChainSize() + 1;
```

Comment:
For me name "period_" sounds like it represents pbft period, but actually this variable does not represent current pbft period (it could be already + 1) because chain is updated before we call cleanupVotesByPeriod. It represents last period for which we cleaned votes so maybe something like `votes_cleanup_last_period_` would be more intuitive what it actually holds ?

Reply:
Renamed